config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  # verb "path", to: "controller#action"
+  get "/users/current", to: "users#current", as: "current_user"
+  resources :works
+  resources :users
```

Comment:
Do you need all the routes for users?

```suggestion
  resources :users, only: [:index, :show]
```